integrations/gitlab/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 <!-- towncrier release notes start -->
 
+0.1.34 (2023-12-12)
+
+### Improvements
+
+- Added support for system hooks (#5220)
```

Comment:
Can you please be a bit more verbose?
When using a port issue please prefix it with PORT-